packages/Interface/src/DTK_C_API.h

```diff
@@ -149,6 +222,17 @@ extern void DTK_finalize();
 
 /**@}*/
 
+/** \brief Get DTK error message.
```

Comment:
Comment is duplicated after the declaration of `DTK_Error`.
Make sure we look at the documentation generated with Sphinx before merging.